lib/signed-xml.js

```diff
@@ -465,10 +465,7 @@ SignedXml.prototype.validateReferences = function(doc) {
           break;
         }
       }
-    
-      if(hasNonExcC14nTransform){
```

Comment:
Given that we no longer need this check anymore, you could remove lines lines 457-467 as well. The commens in 452-455 should be similarly changed